crud.py

```diff
@@ -0,0 +1,49 @@
+from sqlalchemy.orm import Session
+
+import models
+import schemas
+
+
+def get_all_authors(skip: int, limit: int, db: Session):
+    return db.query(models.Author).offset(skip).limit(limit).all()
+
+
+def get_all_books(db: Session, skip: int = 0, limit: int = 5, author_id: int = None):
+    queryset = db.query(models.Book)
+
+    if author_id:
+        queryset = queryset.filter(models.Book.author_id == int(author_id))
+
+    queryset = queryset.offset(skip).limit(limit).all()
+
+    return queryset
+
+
+def get_autor_from_db(db: Session, author_id: int):
```

Comment:
There is a typo in the function name `get_autor_from_db`. It should be `get_author_from_db` to match the correct spelling of 'author'. This will also require updating any references to this function elsewhere in the code.